lib/text-editor-view.coffee

```diff
@@ -0,0 +1,47 @@
+{View} = require 'atom-space-pen-views'
+
+module.exports =
+class TextEditorView extends View
+  # The constructor for setting up an `TextEditorView` instance.
+  constructor: (params={}) ->
+    {mini, placeholderText, attributes, editor} = params
+    attributes ?= {}
+    attributes['mini'] = mini if mini?
+    attributes['placeholder-text'] = placeholderText if placeholderText?
+
+    if editor?
+      @element = atom.views.getView(editor)
```

Comment:
Duplicating this file is no bueno. TODO: update atom-space-pen-views to allow for the `editor` param.